File: C++/7-continue.cpp
#include "stdafx.h"
#include <iostream>
#include "conio.h"

using namespace std;

void Continue()
{
	for (int i = 0; i < 5; i++)
	{
		cout << "Hello ";
		if (i != 4) continue;
		cout << "World!" << endl;
	}

	_getch();
}

File: Java/src/7-continue.java
class Continue {
    public static void main(String[] args) {
        int i = 0;
        label1:
        while (i < 10) {
            i++;
            System.out.println("Hello");
            if (i == 5) continue label1;
            System.out.println("World");
        }
    }
}
